pkgs/applications/audio/listen1/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, fetchurl, appimageTools, makeWrapper }:
+
+let
+  pname = "listen1";
+  version = "2.31.0";
+  name = "listen1-${version}";
+  nameExecutable = pname;
+  src = fetchurl {
+    url = "https://github.com/listen1/listen1_desktop/releases/download/v${version}/listen1_${version}_linux_x86_64.AppImage";
+    name = "listen1-${version}.AppImage";
+    sha256 = "9d80ca7b157356ec0f9affde2bd701d280128171196eb3ebcea3eee4e1e4e8d4";
```

```suggestion
    hash = "...";
```

Please use SRI hash format.